app/src/js/utils/format.js

```diff
@@ -105,21 +105,15 @@ const getIndicator = (prop) => {
       indicator.text = prop;
       break;
     case 'Failed':
-      indicator.color = 'failed';
-      indicator.text = prop;
-      break;
     case 'Running':
-      indicator.color = 'orange';
+      indicator.color = prop.toLowerCase()
```

Comment:
Sorry, I didn't look at this closely enough. This isn't an improvement. I didn't realize there were other indicator.colors that weren't the property name either.  I'd revert this, you should tell me when I'm making bad recommendations, I won't take it personally.

Reply:
It especially won't work since `false` is a possibility.  Ugh, I'm sorry again.  I was going to quick write up what I meant, but obviously it wasn't a fully formed thought.